etl/steps/data/garden/who/2024-04-08/polio.meta.yml

```diff
@@ -0,0 +1,152 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Global Health
+        - Eradication of Diseases
+#  surveillance_sort: &surveillance_sort
+#    - Inadequate screening and testing
+#    - Inadequate screening
+#    - Inadequate testing
+#    - Adequate screening and testing
+#    - Low risk
+#    - nan
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+tables:
+  polio:
+    variables:
+      afp_cases:
+        title: Acute Flaccid Paralysis (AFP) cases
+        description_short: Number of new cases of acute flaccid paralysis (AFP) reported in a given year.
+        unit: cases
+        display:
+          numDecimalPlaces: 0
+      afp_cases_per_million:
+        title: Acute Flaccid Paralysis (AFP) cases per million
+        description_short: Number of new cases of acute flaccid paralysis (AFP) reported in a given year per million people.
+        unit: cases per million
+        display:
+          numDecimalPlaces: 1
+      non_polio_afp_rate:
+        title: Non-Polio acute flaccid paralysis (afp) rate
+        description_short: The number of cases of acute flaccid paralysis (AFP) per 100,000 children aged 15 or under, not attributed to polio, that were detected and reported to the WHO.
+        description_key: ["Acute flaccid paralysis (AFP) surveillance is the gold standard for detecting cases of poliomyelitis.", "At least one case of non-polio AFP should be detected annually per 100,000 population aged less than 15 years. In endemic regions, to ensure even higher sensitivity, this rate should be two per 100 000."]
```

Comment:
Why not using "- Acute ..." for each key item instead of creating a [] list? to improve readability.

Reply:
Ah I was trying to placate VScode telling me the schema was expecting an array, but now I can see there are better ways to do this.